WebRoot/js/ajax/dwt/widgets/DwtMessageDialog.js

```diff
@@ -109,9 +109,6 @@ DwtMessageDialog.TITLE[DwtMessageDialog.WARNING_STYLE] = AjxMsg.warningMsg;
 DwtMessageDialog.TITLE[DwtMessageDialog.PLAIN_STYLE] = AjxMsg.infoMsg;
 
 DwtMessageDialog.ICON = {};
-DwtMessageDialog.ICON[DwtMessageDialog.CRITICAL_STYLE] = "Critical_32";
```

Comment:
Yes, in `_base4` theme, we are removing icons for dialogs altogether as per UX. Also if some skin wants to show icons, they can declare icons into object `DwtMessageDialog.ICON`, and it will render as here: https://github.com/Zimbra/zm-ajax/blob/develop/WebRoot/js/ajax/dwt/widgets/DwtMessageDialog.js#L147

Also if you feels we can keep empty object properties like `DwtMessageDialog.ICON[DwtMessageDialog.WARNING_STYLE] = ""`, but I don't feel it good way. 

Let me know your suggestions.